### DISCUSSION TITLE
Firefox Extension

### DISCUSSION THREAD

**Discussion by serban-mihai** [thetarnav/solid-devtools#211] [General]
Is there any plan also to port the Chrome extension into Firefox?
I know it's not amongst the most used browsers, but some developers (including me) prefer to use it.

**Comment by thetarnav** [comment]
Definitely!
I preferred to focus only on chrome until the project is more mature, and I think that soon I'll have to look into that.
The tool that I'm using for bundling the extension ([a crxjs vite plugin](https://github.com/crxjs/chrome-extension-tools)) is mainly targeting chrome. Not sure how easy it is to make it work with firefox APIs.
